tests/dump-leaf-to-not-verbose.t

```diff
@@ -38,4 +38,12 @@ Dump an example certificate (example-leaf.crt)
   \tlocalhost (esc)
   IP Addresses:
   \t127.0.0.1, ::1 (esc)
+  Warnings:
```

Comment:
Since we'll be including the error level (WARN vs ERROR), I think we should change "Warnings" to "Lints"